README.md

```diff
@@ -1 +1,15 @@
-# CoNLL-U_cycles
\ No newline at end of file
+# CoNLL-U_cycles
+## Summary
+This program takes a CoNLL-U file and finds dependency cycles in its dependency graph.
+
+## What is a dependency cycle?
+
+A dependency cycle occurs when node N depends on the value of node N, either indirectly through a chain of dependencies that loop back or directly by listing itself as a dependency.
+
+## Running
+
+```bash
+python3 main.py file.txt
+```
+
+This will parse a file.txt in the CoNLL-U format, and list cycles.
```

Comment:
Below this, add the two examples in a 'Testing' section. What happens when you run it on `test/test_good.txt`, and why? What about `test/test_cycles.txt`? (This will help clarify what dependency cycles are and also provide testing instructions.)